deployment/playbooks/roles/aws/templates/env.j2

```diff
@@ -0,0 +1,17 @@
+DJANGO_DEBUG="False"
+HOUNDIGRADE_AWS_AVAILABILITY_ZONE="{{ec2_asg_az}}"
+HOUNDIGRADE_AWS_AUTOSCALING_GROUP_NAME="{{ec2_autoscaling_group.auto_scaling_group_name}}"
+HOUNDIGRADE_ECS_CLUSTER_NAME="{{ecs_cluster.cluster.clusterName}}"
```

Comment:
Should this and the lines above it be deleted since we no longer use these env vars in the app?

Actually, now that I think about it, is this env file just an artifact from when we were hoping to put in the PVC?

Reply:
Yes, that is the env file template.

Reply:
`.j2` == jinja

Reply:
Even if we keep the template in hopes that "future us" will find a way to make it relevant, these actual values (`DJANGO_DEBUG`, `HOUNDIGRADE_AWS_AVAILABILITY_ZONE`, `HOUNDIGRADE_AWS_AUTOSCALING_GROUP_NAME`, and `HOUNDIGRADE_ECS_CLUSTER_NAME`) appear not to be used for deployments, and if that's correct, I think they should be culled from the templated env file. We do load these value names in `base.py` (like [here](https://github.com/cloudigrade/cloudigrade/blob/959d956f4440513542c37d581fd7cb0a1ecce1dc/cloudigrade/config/settings/base.py#L124)), but then we immediately overwrite them with values in `prod.py` [here](https://github.com/cloudigrade/cloudigrade/pull/814/files#diff-77fcf72288500c6f144a8a8c99afd446dde68815c69f27edd3d7358389a80d1bR14).

I assume this generated env file would only be used in "real" deployments, not locally, which would mean these values will _always_ get overwritten by their `prod.py` counterparts. Or do you envision a way that this file could be generated and used for local development where `local.py` _doesn't_ overwrite the values from the env vars/file? Or have I misread the intention of this file?